packages/terraform-modules-public/aws/testnet/modules/ami/outputs.tf

```diff
@@ -1,5 +1,5 @@
 output ami_ids {
   value = {
-    ubuntu_18_04 = data.aws_ami.ubuntu.id
+    ubuntu_20_04 = data.aws_ami.ubuntu.id
```

Comment:
Should we name this something like `ubuntu_latest_lts` so that we don't have to change modules using this each time this changes?

Reply:
Yep, I had the same thought. I'll do that